be/src/pipeline/exec/partitioned_aggregation_sink_operator.cpp

```diff
@@ -244,8 +244,16 @@ Status PartitionedAggSinkLocalState::revoke_memory(RuntimeState* state) {
             }
         }
     }};
+
+    auto execution_context_ = state->get_task_execution_context();
+    _shared_state_holder = _shared_state->shared_from_this();
+
     status = ExecEnv::GetInstance()->spill_stream_mgr()->get_async_task_thread_pool()->submit_func(
-            [this, &parent, state] {
+            [this, &parent, state, execution_context_] {
```

Comment:
Has to use value to copy the variable, not use reference in lambda object.
Because the referenced object maybe destroyed.
You could refer this PR 
https://github.com/apache/doris/pull/32132/files